src/groovy/org/pih/warehouse/core/EventCode.groovy

```diff
@@ -24,11 +24,28 @@ enum EventCode {
     DELIVERED,
     RECEIVED,
     PARTIALLY_RECEIVED,
-    CANCELLED
+    CANCELLED,
+    WAITING_FOR_APPROVAL,
```

Comment:
I'm wondering if we could come up with a more generic term for this like SUBMITTED. Or maybe we need both SUBMITTED and something like PENDING_APPROVAL.

Reply:
If it were up to me, I'd distinguish them. Having said `PENDING_APPROVAL`, do you mean that I should change the naming from `WAITING_FOR_APPROVAL` to `PENDING_APPROVAL`, or it was just an open suggestion?

Reply:
Changed to `PENDING_APPROVAL`, but left the translation as "Waiting for approval". Additionally - added translations for all of the EventCodes, since there were not any.